app/lib/LevelSetupManager.coffee

```diff
@@ -68,7 +68,7 @@ module.exports = class LevelSetupManager extends CocoClass
      @onInventoryModalPlayClicked()
      return
 
-    if @level.isType('course', 'course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
+    if @level.isType('course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
```

Comment:
This should probably be controlled by the classroomItems flag

Reply:
Also, please check if this flag needs to be checked for other conditions here as well like line 67 and line 75